docs/course/index.md

```diff
@@ -133,7 +133,7 @@ This file specifies basic information about the course:
 ??? note "Format specification for `infoCourse.json`"
 
     ```json
-    --8<-- "apps/prairielearn/src/schemas/schemas/infoCourse.json"
+    --8<-- "apps/prairielearn/src/schemas/infoCourse.json"
```

Comment:
This file no longer exists but the docs still build without error. That worries me a lot; can you check our documentation config to see if we can get it to fail loudly when a file referenced like this doesn't exist?

Reply:
I'll take a look at this on Monday.

Reply:
Changed in #11186.